### PR TITLE
Contours for S/M/L NIOSH headforms

### PR DESCRIPTION
**Pull request by Burhan-Q**
Closes issues #47, #46, and #45

@darraghbr should review for completeness, @mmroden added for merging (if needed)

### CODE REVIEW

File: Face_contours/STEP/full_face/_STEP-full-face_model_notes.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+2020-02-06
```
Comment (darraghbr) [inline Face_contours/STEP/full_face/_STEP-full-face_model_notes.txt:1]: Is this date correct?

File: Face_contours/STEP/_STEP-face_surface_model_notes.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+2020-02-06
```
Comment (darraghbr) [inline Face_contours/STEP/_STEP-face_surface_model_notes.txt:1]: Is this date correct?

**Comment by darraghbr** [comment]
What is the original source for these contours?

File: Face_contours/STEP/_STEP-face_surface_model_notes.txt (hunk 2 above)
Comment (Burhan-Q) [reply Face_contours/STEP/_STEP-face_surface_model_notes.txt:1]: @darraghbr nope, you're 100% correct. I mistyped

File: Face_contours/STEP/full_face/_STEP-full-face_model_notes.txt (hunk 1 above)
Comment (Burhan-Q) [reply Face_contours/STEP/full_face/_STEP-full-face_model_notes.txt:1]: @darraghbr nope, you're 100% correct. I mistyped

**Comment by darraghbr** [comment]
Please add references to where the original data is pulled from, if you can do it in the Chicago style and add in when the data was accessed.

**Comment by darraghbr** [comment]
Reviewed contours against original information pulled from NIOSH (https://www.cdc.gov/niosh/data/datasets/rd-10130-2020-0/default.html) 

image

Contours look very close, only varying around the base of the nose.

**Review by darraghbr — CHANGES_REQUESTED** [review]
Please see my previous comments

**Comment by Burhan-Q** [comment]
> Please see my previous comments

@darraghbr I used the IGES headform models from the same link. As far as the specific contour, the process is described below:

Beginning with the NIOSH large 3D headform, the initial design was created on a 2D plane using a B-spline for the inner profile, relative to the headform not using any explicit geometric relations. The spline positioning and shape was adjusted to mimic the coverage of existing FFRs. A copy of the profile was then projected 4 mm radially outward in the same plane to create the outer profile. These two B-splines were joined with vertical lines at either end to create a closed contour. The contour was projected from the 2D plane onto the 3D surface of the Large NIOSH Headform from the Principal Component Analysis Study, forming the face contact surface. The surface was then mirrored about the center plane.

**Comment by darraghbr** [comment]
Apologies, I was unclear, the contours and files are fine. The change I am requesting is them to be formally referenced.

**Comment by mmroden** [comment]
"Advanced Headforms for Evaluating Respirator Fit", Michael Bergman, MS; Ziqing Zhuang, PhD; Ronald E. Shaffer, PhD . July 25, 2013.  Referenced from https://blogs.cdc.gov/niosh-science-blog/2013/07/25/headforms/ on Feb 13, 2021.

"NIOSH Anthropometric Data and ISO Digital Headforms" Updated on Updated April 23, 2020.  NIOSH Dataset RD-10130-2020-0. Referenced from https://www.cdc.gov/niosh/data/datasets/rd-10130-2020-0/default.html on Feb 13, 2021

Additional references: 

1.     Zhuang, Z.Q., B. Bradtmiller, and R.E. Shaffer: New respirator fit test panels representing the current US civilian work force. Journal of Occupational and Environmental Hygiene 4(9): 647-659 (2007).
2.     Zhuang, Z., S. Benson, and D. Viscusi: Digital 3-D headforms with facial features representative of the current US workforce. Ergonomics 53(5): 661-671 (2010).
3.     Hanson, D., R. Bergs, Y. Tadesse, V. White, and S. Priya: Enhancement of EAP Actuated Facial Expressions by Designed Chamber Geometry in Elastomers. Proc. SPIE’s Electroactive Polymer Actuators and Devices Conf., 10th Smart Structures and Materials Symposium, San Diego(2006).
4.     Hanson, D., and V. White: Converging the Capabilities of ElectroActive Polymer Artificial Muscles and the Requirements of Bio-inspired Robotics. Smart Structures and Materials 2004: Electroactive Polymer Actuators and Devices (EAPAD), edited by Yoseph Bar-Cohen, Proceedings of SPIE Vol. 5385 (SPIE, Bellingham, WA), 2004.
5.     Bergman, M., Z. Zhuang, R.J. Wande, D. Hanson, B. Heimbuch, M. McDonald et al.: Development of an Advanced Respirator Fit Test Headform. Journal of Occupational and Environmental Hygiene In Press(2013).
6.     Zhuang, Z., M. Bergman, M. Joseph, and R. Shaffe: Human Correlation Fit Testing of a Static Advanced Headform. Presentation at the NPPTL Stakeholder Meeting on Respiratory Protection for Healthcare Workers, Atlanta, GA June 18, 2013 (2013).

**Comment by Burhan-Q** [comment]
@darraghbr I added the information @mmroden to a text file into this PR

**Review by darraghbr — APPROVED** [review]
Looking good.

**Review by mmroden — APPROVED** [review]
Darryl approves

**Pull request merged**